src/averageAges.js

```diff
@@ -55,10 +61,29 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
```

Comment:
The condition in the filter function should check if the person's sex is 'f' and there is at least one person who mentions her as a mother. The current condition is incorrect.